packages/app/src/cli/models/extensions/specifications/web_pixel_extension.ts

```diff
@@ -10,10 +10,24 @@ const BUNDLE_SIZE_LIMIT = BUNDLE_SIZE_LIMIT_KB * kilobytes
 
 const dependency = '@shopify/web-pixels-extension'
 
+enum DataSaleConfig {
+  SellsData = 'sells_data',
+  DoNotSellData = 'do_not_sell_data',
+  SupportsLduDowngrade = 'supports_ldu_downgrade',
+}
+
 const WebPixelSchema = BaseSchema.extend({
   runtime_context: zod.string(),
   version: zod.string().optional(),
   configuration: zod.any(),
+  privacy_settings: zod
+    .object({
+      analytics: zod.boolean().optional(),
+      preferences: zod.boolean().optional(),
+      marketing: zod.boolean().optional(),
+      sale_of_data: zod.string().optional(),
```

Comment:
```ts
sale_of_data: zod.enum(['sells_data', 'do_not_sell_data', 'supports_ldu_downgrade']).optiona()
```
This will ensure only those values are accepted, no need to do any extra validation